modules/nodes-cluster-overcommit-configure-nodes.adoc

```diff
@@ -43,8 +43,8 @@ vm.panic_on_oom = 0
 You can change these settings using:
 
 ----
-$ sysctl -w vm.overcommit_memory=1
-$ sysctl -w vm.panic_on_oom=0
+$ sysctl -w vm.overcommit_memory=0
+$ sysctl -w vm.panic_on_oom=1
```

Comment:
panic_on_oom should be set to 0... 0 does not cause the kernel to core.

Reply:
Would there be a situation where the user might change to 1?

Reply:
`panic_on_oom=1` is not likely with k8s nodes since it causes the kernel to panic and halt.